CODE_OF_CONDUCT.md

```diff
@@ -34,7 +36,7 @@ This Code of Conduct applies both within project spaces and in public spaces whe
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at contact@grafana.com. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the Grafana Labs conduct team at conduct@grafana.com. The conduct team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```

Comment:
```suggestion
Report instances of abusive, harassing, or otherwise unacceptable behavior by contacting the Grafana Labs conduct team at conduct@grafana.com. The conduct team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team must maintain confidentiality regarding the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```